python/nano/setup.py

```diff
@@ -167,7 +167,10 @@ def setup_package():
 
     inference_requires = ["onnx==1.12.0",
                           "onnxruntime==1.12.1",
-                          "onnxruntime-extensions==0.4.2; platform_system!='Darwin'",
+                          "onnxruntime-extensions==0.4.2; platform_system!='Darwin' and \
+                          python_full_version<'3.10.0'",
+                          "onnxruntime-extensions==0.7.0; platform_system!='Darwin' and \
+                          python_full_version>='3.10.0'",
```

Comment:
should we just bump to 0.7.0 for all python versions? I think it still support py37 ~ py39 https://pypi.org/project/onnxruntime-extensions/#files

Reply:
Done